packages/account/src/signingKeychain.ts

```diff
@@ -194,12 +195,14 @@ const create = (
 				): Observable<HardwareSigningKeyT> =>
 					hardwareWallet.makeSigningKey(hdPath),
 			),
-			map((hardwareSigningKey: HardwareSigningKeyT) =>
-				SigningKey.fromHDPathWithHWSigningKey({
+			map((hardwareSigningKey: HardwareSigningKeyT) => {
+				const signingKey = SigningKey.fromHDPathWithHWSigningKey({
 					hdPath,
 					hardwareSigningKey,
-				}),
-			),
+				})
+				setActiveSigningKey(signingKey)
```

Comment:
I dont like hiding this quite important side effect, so can we maybe at least add it as a boolean option flag to `input: DeriveHWSigningKeyInput`, naming it: `alsoSwitchTo` like we have done with all other account derivation functions?

Reply:
done